lexos/processors/prepare/scrubber.py

```diff
@@ -787,11 +642,98 @@ def load_cached_file_string(cache_folder: str, filename: str) -> str:
         return ""
```

Comment:
`cache_file_string`, and `load_cache_file_string` can be a wrapper of `cache_to_disk` and `load_cache_from_disk`

This wrapper can be used to provide type hinting